main.md

```diff
@@ -48,7 +48,7 @@ used to carry fine-grained authorization data in OAuth messages.
 
 # Introduction {#Introduction}
 
-The OAuth 2.0 authorization framework [@!RFC6749] defines the parameter `scope` that allows OAuth clients to
+The OAuth 2.0 authorization framework [@!RFC6749] defines the "scope" parameter that allows OAuth clients to
```

Comment:
```suggestion
The OAuth 2.0 authorization framework [@!RFC6749] defines the `scope` parameter that allows OAuth clients to
```

Reply:
the review specifically suggested that scope be quoted, which the tick syntax doesn't do.

Reply:
We're putting other parameter names like `authorization_details` in ticks, and so this should probably be consistent since it's the same level of item, thus my change. The transformation to plain text is :supposed: to make that a quoted string in output, but I've seen that fail in some weird ways depending on the version of the document toolchain, so we might be seeing an artifact based on which version the reviewer was referencing.

Reply:
Yes and whatever toolchain is doing this one is not https://datatracker.ietf.org/doc/html/draft-ietf-oauth-rar-16 quoting. I was just trying to get though this review with as little hassle as possible. So used some quotes. But I don't actually care.